src/domain/entities/SharedObject.ts

```diff
@@ -1,8 +1,6 @@
-import { NamedRef } from "./Ref";
+import { SharingRule } from "@eyeseetea/d2-ui-components";
```

Comment:
You never have to take entities of your domain from outside, instead create a new one:
```typescript
export type SharingSetting = { id: Id; displayName: string; access: string; name: string };

```